src/DynamoCoreWpf/Services/IconServices.cs

```diff
@@ -54,7 +54,10 @@ internal IconWarehouse(Assembly resAssembly)
             if (resAssembly != null)
             {
                 resourceAssembly = resAssembly;
-                assemblyName = resAssembly.GetName().Name.Split('.').First();
+
+                // Remove ".customization" suffix and remove middle dots
+                var temp = resAssembly.GetName().Name.Split('.');
+                assemblyName = String.Join("", temp.Take(temp.Length - 1));
```

Comment:
For easier understanding, can you cite an example here in the comment? For example:

```csharp
// "Name" can be "Some.Assembly.Name.customization" with multiple dots, 
// we are interested in removal of the "customization" part and keep the full name.
```